CMakeLists.txt

```diff
@@ -105,18 +105,11 @@ if(CCACHE_FOUND)
     set(CMAKE_CXX_COMPILER_LAUNCHER ccache)
 endif()
 
-if(WIN32)
-    add_compile_definitions(UNICODE)
-    add_compile_definitions(_WIN32_WINNT=0x0602)
```

Comment:
Why are these moved?

Reply:
Now these definitions are added to targets that require them and not globally.